components/landing/TheHeroMoment.vue

```diff
@@ -90,16 +90,18 @@ export default class extends Vue {
 
   &__description {
     @include type-style('body-long-01');
-    color: $white-text-01;
+    color: $gray-60;
+    max-width: 4.75 * $column-size-large;
```

Comment:
Regarding `max-width`, I'd prefer if we use some multiple of `$column-size-large`. I'd say it is not 6 but can you confirm what's the correct value with @JRussellHuffman, please?

Reply:
Yes, he thinks it looks better with 5 columns, I will change it